conf/cmi/core.entity_view_display.node.webform.default.yml

```diff
@@ -43,6 +42,5 @@ content:
     region: content
 hidden:
   field_form_code: true
-  field_url_to_form_service: true
```

Comment:
Changes to this file should be reverted.

Reply:
done.